### ISSUE TITLE
Cannot send message without text in it

### ISSUE THREAD

**Issue by thomascube** [thomascube/roundcube-trac-import#62]
**Reported by theritz on 25 May 2007 08:08 UTC**

If you compose a message with nothing in it or just one or more attachements and no text Roundcube will not send it. You can push the Send button all you want but it will stay in the compose message screen.

Migrated-From: http://trac.roundcube.net/ticket/1484397

**Comment by thomascube** [comment]
**Comment by thomascube on 25 May 2007 21:26 UTC**
Works for me. Check your error logs for SMTP server responses.

**Comment by thomascube** [comment]
**Comment by seansan on 26 May 2007 07:53 UTC**
See attachment - I also recognize the error the user has, I have also seen this before.

- Create a message, enter recipient, subject, click send. ERROR as in attachment.

browser IE6, Windows XP

**Comment by thomascube** [comment]
**Comment by kmn on 27 May 2007 03:14 UTC**
Getting the same problem, but no error message. The mail just sits idle without getting sent. Even adding an attachment also does not work.

Using XP SP2 OS, IE6 in the client machine.

**Comment by thomascube** [comment]
**Comment by theritz on 4 Jun 2007 13:06 UTC**
When sending an empty message from IE 7.0.5730.11 and Win XP SP2 I get to see the error I put in the err.jpg attachement. When sending with text I get to see no error at all and the message is sent.

**Comment by thomascube** [comment]
**Comment by thomascube on 5 Jun 2007 19:47 UTC**
Found! Must have missed something when removing tinyMCE editor.
```
Index: program/js/app.js
===================================================================
--- program/js/app.js   (revision 577)
+++ program/js/app.js   (working copy)
@@ -1680,7 +1680,7 @@
       }
 
     // check for empty body
-    if ((input_message.value=='')&&(tinyMCE.getContent()==''))
+    if (input_message.value=='')
       {
       if (!confirm(this.get_label('nobodywarning')))
         {
```

**Comment by thomascube** [comment]
**Comment by thomascube on 16 Aug 2007 08:07 UTC**
Fixed in trunk ([31d9efd9])
